julia/appveyor.yml

```diff
@@ -0,0 +1,39 @@
+environment:
```

Comment:
Would you be able to maintain this CI system?

Reply:
👌

Reply:
This is only for Windows, right? I think we will run a hybrid mode in the beginning. At the moment, we're not able to perform any changes to the Windows slaves, so this solution would be the best for now. But if you don't mind, we'll make AppVeyor non-required. In the long term, I'd like to move this to jenkins and I'd appreciate your assistance there.